Source/Orts.Formats.Msts/CabViewFile.cs

```diff
@@ -311,6 +311,14 @@ public enum CABViewControlUnits
         LBS
     }
 
+    public static class DiscreteStates
+    {
+        public const string Lever = "lever";
+        public const string TwoState = "twostate";
+        public const string TriState = "tristate";
+        public const string MultiState = "multistate";
+    }
```

Comment:
I think an enumerated value would have been better here.
And we don't need to use this const string inside the first parameter TokenProcessor constructor below. It is less readable than before.

Reply:
Thanks, Cedric. I'll try it like that.